vars/applier.groovy

```diff
@@ -1,36 +1,52 @@
 #!/usr/bin/env groovy
 
+// orginial from https://github.com/redhat-cop/pipeline-library/blob/master/vars/applier.groovy
+
 class ApplierInput implements Serializable{
-    String secretName = ""
-    String clusterURL = ""
-    String inventoryPath = ""
-    String requirementsPath = ""
-    String skipTlsVerify = false
-    String rolesPath = "galaxy"
-    String applierPlaybook = "galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml"
+    String inventoryPath          = ''
+    String requirementsPath       = ''
+    String ansibleRootDir         = ''
+    String rolesPath              = 'roles'
```

Comment:
please change to 'galaxy'. that's been the applier naming standard since its not simply a role.

Reply:
i'll change it back, but it didn't work for us with client when using it with galaxy, ansible couldn't find the role, only thing that worked was changing that value to roles.

Reply:
@etsauer done.